docs/release-notes/highlights-1.1.0.asciidoc

```diff
@@ -65,6 +65,58 @@ The concept of `global` and `namespace` operator roles no longer exists, and as
 
 Container ports of managed resources are now named according to the protocol (`https` for TLS enabled ports and `http` for TLS disabled ports). Previously they were named `http` regardless of the protocol being used. If you have any configurations that refer to the container ports by name (Ingress configurations, for example), review those to ensure that they use the correct name. One option may be to temporarily change to targeting the port number (e.g. `9200` for Elasticsearch), rather than the name, before updating ECK. Once the container port is updated to the new name, you can change to targeting the port name again.
 
+[float]
+[id="{p}-110-breaking-readiness-probe"]
+==== Readiness probe change for Kibana and APM Server
+
+Default readiness probes generated by the operator for Kibana and APM Server resources have changed from HTTP GET actions to exec actions. This is to accommodate environments where host-to-pod communication is restricted for security reasons. This change will affect ingress resources that are used to expose Kibana or APM Server instances as some ingress controllers re-purpose the pod readiness probe as a health check. If you are unable to access Kibana or APM Server instances through the ingress after upgrading to ECK 1.1.0, override the readiness probe definitions as follows:
```

Comment:
```suggestion
Default readiness probes generated by the operator for Kibana and APM Server resources have changed from HTTP GET actions to exec actions. This is to accommodate environments where host-to-pod communication is restricted for security reasons. This change will affect ingress resources that are used to expose Kibana or APM Server, as some ingress controllers re-purpose the pod readiness probe as a health check. If you are unable to access Kibana or APM Server through the ingress after upgrading to ECK 1.1.0, override the readiness probe definitions as follows:
```

Reply:
Small suggestions: adding a `,` and removing "instances" since I'm not sure it is necessary.

I think we want to recommend upgrading to 1.1.1 as the first option, the second option is to override it as described below

Reply:
Do we want to refer to a future release from a past release? It feels a bit strange to me. Furthermore, I think we want to get this doc change published immediately so that means the 1.1.1 upgrade option would not be available for a little while longer.

Reply:
That's a good point. @alaudazzi do you know how we handle this in other parts of the docs? If not then I am good with your plan Charith